test/cases/api.sh

```diff
@@ -895,6 +895,15 @@ function verifyInAWSS3() {
   local S3_URL
   S3_URL=$(echo "$UPLOAD_OPTIONS" | jq -r '.url')
 
+  # Tag the resource as a test file
+  local S3_FILENAME
+  S3_FILENAME=$(echo "${S3_URL}" | grep -oP '(?<=/)[^/]+(?=\?)')
```

Comment:
I'm not sure about this regex if we can rely on it to work if some changes to the URL were to be made. Is it not possible to get it somewhere from the `$UPLOAD_OPTIONS` with the use of `jq`?

Reply:
We use the same pattern on line [188](https://github.com/osbuild/osbuild-composer/blob/649977c6c6076c557594d5e42b985afc2b9ca8f4/test/cases/api.sh#L188). But I could look into it and get the name outside of the url.

Reply:
Oh ok I didn't see it. I don't think you need to change it if it's used already somewhere.